docs/main/README.md

```diff
@@ -178,6 +178,13 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
     oc apply -n orchestrator -f https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/main/config/samples/_v1alpha1_orchestrator.yaml
     ```
 
+### Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfills this reliability need.
```

Comment:
Does it mean that the user has other options?

Reply:
Yes https://knative.dev/docs/eventing/brokers/broker-types/

Reply:
i think it may be nice to add that link